lib/config/go_routes/go_router.dart

```diff
@@ -432,13 +433,15 @@ abstract class AppRoutes {
                         );
                       }
                       return CupertinoPage(
+                        name: '/room',
```

Comment:
why we need this?

Reply:
we need a name so that later we can use popUntil method to back to Chat screen